gpbft/gpbft.go

```diff
@@ -262,7 +267,7 @@ func (i *instance) ReceiveMany(msgs []*GMessage) error {
 	// Received each message and remember which rounds were received.
 	roundsReceived := map[uint64]struct{}{}
 	for _, msg := range msgs {
-		if err := i.receiveOne(msg); err != nil {
+		if _, err := i.receiveOne(msg); err != nil {
```

Comment:
This path is reached only once per instance, so I don't think it's worth the check here. But I can accumulate a stateChanged here too if requested.

Reply:
We're limited to `maxLookaheadRounds * powerLookback` anyways (except for justified messages). So I'd say this is fine.

Reply:
Although... it should be trivial to just not add this round to `roundsReceived`, right?

Reply:
Yes,done